docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,246 @@
+|----------------------	|------------------------------------------------------------------------------------------------------------------------------------	|-----------------	|------------------	|---------------------------------------------------------------------------------------------------	|
+| Platform Region (**PR**) 	| string, subaccount regions as defined in [Regions for the Kyma Environment](https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment) 	| true            	| false            	| hyperscalerType: `<providerType>_<PR>` or hyperscalerType: `<providerType>_<PR>_<HR>` if used with **HR** 	|
+| Hyperscaler Region (**HR**)  	| string, cluster region as defined in [Regions for the Kyma Environment](https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment)   	| true            	| false            	| hyperscalerType: `<providerType>_<HR>` or hyperscalerType: `<providerType>_<PR>_<HR>` if used with **PR**	|
+| EU Access (**EU**)       	| no value                                                                                                                           	| false           	| true             	| euAccess: true                                                                                    	|
+| Shared (**S**)           	| no value                                                                                                                           	| false           	| true             	| shared: true                                                                                      	|
+
+> [!NOTE] 
+> * Input/Output attributes - true when the attribute can occur in the input/output attributes section (left part) of a rule entry (see [Rule Format](#rule-format) section).
+> * Modified SecretBinding search labels - lists modifications of search labels that the rule entry containing the attribute adds to search labels if triggered. 
+
+## Uniqueness and Priority
+
+Only one rule can be triggered. If more than one rule entry matches the request, only one is selected and applied. The process of selecting the best matching rule is based on rule uniqueness and priority.
+Rule entry uniqueness is determined by its plan and input parameters' values (identification attributes) not including input parameters with `*`. 
+Output parameters are not taken into account when establishing rule entry uniqueness. For example, the following rule fails on startup because both rules can match every Kyma runtime:
+
+```
+hap:
+  rule: 
+    - gcp 
+    - gcp -> S                      # invalid entry, output attributes does not take part into uniqueness check
+    - gcp(PR=*)                     # invalid entry, both can be applied to the same Kyma Runtime
+    - gcp(HR=europe-west3)          # valid entry, new **HR** attribute makes the rule unique
```

Comment:
It seems you can't use markdown inside code samples. Please change \*\*HR** to HR.